### PR TITLE
Chloe/create map

### PR DESCRIPTION
**Pull request by chloelu29**
convert address to coordinates 
pin onclick renders corresponding info from database on post and expanded post

### CODE REVIEW

File: client/actions/actions.js [hunk 1]
```diff
@@ -33,10 +33,42 @@ export const getUsername = (username, password) => (dispatch) => {
...
+export const changeActivePost = (incident_id) => (dispatch, getState) =>{
+  
   const allIncidents = getState().map.allIncidents;
-  dispatch({ type: types.CHANGE_ACTIVE_POST, payload: [lat, long], allIncidents: allIncidents });
+  dispatch({ type: types.CHANGE_ACTIVE_POST, payload: incident_id, allIncidents: allIncidents });
+  
+};
+
+export const getCoordinates = () => (dispatch) => {
+  axios.get(`api/incidents/location/NY`)
```
Comment (huyqbui) [inline client/actions/actions.js]: if we want the map to display locations in all areas, and not just NY, we can use the `api/incidents` instead
```suggestion
  axios.get(`api/incidents`)
```

File: client/components/Map.jsx [hunk 2]
```diff
@@ -20,28 +22,31 @@ const mapStateToProps = ({map: {viewport, pinLocations}}) => ({
...
+
+                // console.log(e.target.value)
+                props.changeActivePost(el.id)
+
+                }}style={{backgroundColor: 'transparent'}}><img src={'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRCkP7QFhZEIqrhTgIxZIJKmW0zBB50L1Fc6xVI4T7U8cNaNHxEOjxu3Wl578bNT6DHmsY&usqp=CAU'} alt='pin' style={{backgroundColor: 'transparent', height: '50px', width: '50px'}}/></button>
+            </Marker>
+            )
           }
           )}
           <Marker key={0} latitude={40.7128} longitude={-74}>
```
Comment (huyqbui) [inline client/components/Map.jsx]: Will we still need this Marker that has a set latitude and longitude, or was it just for testing?

File: client/reducers/postsReducer.js [hunk 3]
```diff
@@ -2,11 +2,11 @@ import * as types from '../constants/actionTypes';
...
   title: 'Noise Disturbance',
-  location: 'Codesmith HQ',
+  steet_name: 'Codesmith HQ',
   details: 'Reports of excessive hollering and clapping noise',
   time: '6:30pm',
-  imgUrl: 'https://static.frontendmasters.com/assets/teachers/sentance/thumb@2x.jpg',
-  videoUrl: 'https://www.youtube.com/embed/sqAwvpw_FKc',
+  image_url: 'https://static.frontendmasters.com/assets/teachers/sentance/thumb@2x.jpg',
+  video_url: 'https://www.youtube.com/embed/sqAwvpw_FKc',
   expandedPost: false,
```
Comment (huyqbui) [inline client/reducers/postsReducer.js]: Should we change initialState to null values now that we have the map Component working?

**Review by huyqbui — APPROVED** [review]
The comments I marked are mostly for cleanup now that we have the map component working. Great job getting this all to work!

File: client/components/Map.jsx (hunk 2 above)
Comment (chloelu29) [reply client/components/Map.jsx]: just for testing. that's for the original pin. i'll delete that.

File: client/actions/actions.js (hunk 1 above)
Comment (chloelu29) [reply client/actions/actions.js]: yep let's do that

File: client/reducers/postsReducer.js (hunk 3 above)
Comment (Ruzeb) [reply client/reducers/postsReducer.js]: no, the 'post' on the map is always visible, so it needs a default value

Comment (chloelu29) [reply client/reducers/postsReducer.js]: Good point!
just set the post initial state to null and let post not show up the moment page renders

Comment (chloelu29) [reply client/reducers/postsReducer.js]: i got it disappear if the title is a null value. depends on how you guys want the app to be.

Comment (chloelu29) [reply client/reducers/postsReducer.js]: > no, the 'post' on the map is always visible, so it needs a default value

i didn't refresh page when i reply to huy. didn't see ruzeb's post sry.

Comment (Ruzeb) [reply client/reducers/postsReducer.js]: it's actually fine, i thought it would break with nulls but i tested it out and it didnt

**Pull request merged**